### DISCUSSION TITLE
Code Explanations Feature?

### DISCUSSION THREAD

**Discussion by laserbeak** [community/community#38089] [Copilot Conversations] [answered]
Hello,
I’ve just started looking into “AI Coding “ and have been making comparisons between Replit’s Ghostwriter and Copilot.
Does Copilot have the [code explanation](https://docs.replit.com/ghostwriter/explain-code) feature or something similar?

**Comment by david-engelmann** [comment, marked as answer]
It does! [This blog post](https://christianheilmann.com/2022/06/07/github-copilot-explaining-code-and-translating-it-from-one-language-to-another/) has a great deep dive into it! One of the coolest features introduced by Github in a long time. Happy Hackings!

**Reply by laserbeak** [reply]
Oh woooooow! Thank you so much!
Have a great one!

**Reply by laserbeak** [reply]
Awwww... seems like access is limited at the moment :(

**Reply by david-engelmann** [reply]
> Awwww... seems like access is limited at the moment :(

Copilot hasn't been released for the free public yet. I believe some active members were given beta access

**Reply by laserbeak** [reply]
I'm able to use Copilot, just not the add-on for explanations. 
I'm stuck with a dilemma.... be able to use my favorite editor, or use code explanations.
I'd hold my breath for Copilot's version, if I knew when it'd be available.
